Lombiq.Hosting.MediaTheme.Targets/Lombiq.Hosting.MediaTheme.Targets.csproj

```
@@ -0,0 +1,37 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <DefaultItemExcludes>$(DefaultItemExcludes);.git*;node_modules\**</DefaultItemExcludes>
+    <IsPublishable>false</IsPublishable>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Title>Lombiq Hosting - Media Theme Targets</Title>
+    <Authors>Lombiq Technologies</Authors>
+    <Copyright>Copyright © 2023, Lombiq Technologies Ltd.</Copyright>
+    <Description>Lombiq Hosting - Media Theme Targets: Provides automatic media theme recipe and tempalte updating. See the project website for detailed documentation.</Description>
+    <PackageIcon>NuGetIcon.png</PackageIcon>
+    <PackageTags>Lombiq;MediaTheme;MSBuild</PackageTags>
```

I copied these from the Deployer's csproj to make these consistent.

```suggestion
    <PackageTags>OrchardCore;Lombiq;AspNetCore;MSBuild,Media;Targets</PackageTags>
```